### DISCUSSION TITLE
BelongsTo search should be case insentitive with json field

### DISCUSSION THREAD

**Discussion by whitespacecode** [laravel/nova-issues#4770] [Ideas & Feature Requests]
- Laravel Version: 4.13.0
- Nova Version: 4.13.0
- PHP Version: 9.27.0
- Database Driver & Version:
- Operating System and Version: MacOs
- Browser type and version: Chrome

### Description:
I see the issue to search onto json columns is fixed on regular search?
https://github.com/laravel/nova-issues/issues/3303
https://github.com/laravel/nova-issues/issues/509 

The problem still presents when you have a BelongsTo with searchable()
When you want to create a new record the search is still case insensitive. 

Example (Product have a json name field)
```
BelongsTo::make(__('Product'), 'product', Product::class)
                ->searchable()
                ->sortable(),
```

**Comment by crynobone** [comment]
I believe you can come out with a custom relation search class if you need it right now. Since this change the current behavior I'm putting it under feature request for now.
